### PR TITLE
feat(aws-codebuild): add 2x_large compute type option

### PR DESCRIPTION
**Pull request by DmitryNefedov**
As discussed within this feature [request](https://github.com/aws/aws-cdk/issues/5401)
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by DmitryNefedov** [comment]
Hmmm, for some reason even `master` is failing for me...
```
git clone
./install.sh
yarn build
```

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ and let us know if it's not up-to-date (even better, submit a PR with your  corr
   - [Pull Request Checklist](#pull-request-checklist)
   - [Step 1: Open Issue](#step-1-open-issue)
   - [Step 2: Design (optional)](#step-2-design-optional)
-  - [Step 3: Work your Magic](#step-3-work-your-magic)
+  - [Step 3: Work your Magic](#step-3-work-your-magic)S
```
Comment (skinny85) [inline CONTRIBUTING.md]: This change is probably accidental, right?

Comment (DmitryNefedov) [reply CONTRIBUTING.md]: yeah, sorry.
cleaned up.

**Comment by skinny85** [comment]
Thanks @DmitryNefedov !

**Pull request merged**